### PR TITLE
Hetio integration

### PR DESCRIPTION
**Pull request by vincerubinetti**
This PR essentially pulls out the CSS that we want to be global to the entire het.io site.

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -246,7 +242,7 @@ export function transferQueryProps(
 }
 
 // compare two elements as primitives (eg number === number or string ===
-// string) or as arrays if both are arrays
+// string) or as arrays ifesc both are arrays
```
Comment (dongbohu) [inline src/util.js]: Is `ifesc` supposed to be `if`?

**Comment by vincerubinetti** [comment]
He should be. This doesn't really change anything (except for some indentation and incidental things, I think). It's really just pulling out some CSS to be moved to a different file (the `global.css` in het.io).

**Comment by dhimmel** [comment]
> This doesn't really change anything (except for some indentation and incidental things, I think).

Okay then I will pass on reviewing.

**Comment by vincerubinetti** [comment]
> Okay then I will pass on reviewing.

Yeah there's no need.

Also, we'll hold off on merging this until https://github.com/hetio/het.io/pull/7 is also ready to go to minimize the downtime between them.

**Comment by vincerubinetti** [comment]
From the latest commit, here's how it looks build and run locally. All the styles are imported successfully. The only thing missing is the nav bar header and footer (which of course come from the main het.io repo), and it takes the full width of the page because it's not in its final conainer on the het.io site (which is width-limited).

image

**Pull request merged**